### PR TITLE
Support solidus 2 5

### PR DESCRIPTION
**Pull request by DanielePalombo**
In solidus 2.5 the `jquery-ui-datepicker` has been replaced with Flatpickr.
This extension requires a way to set also the time on the sale price.

This PR is going to add the ability to support DateTime using Flatpickr.

### CODE REVIEW

**Comment by DanielePalombo** [comment]
This PR is WIP because I want check that it works in all supported solidus version

File: app/helpers/spree/base_helper_decorator.rb [hunk 1]
```diff
@@ -22,9 +20,21 @@ def supported_currencies_for_sale_price
...
+  def gte_solidus_2_4?
+    gte_solidus?('2.4.0')
+  end
+
+  def gte_solidus_2_5?
+    gte_solidus?('2.5.0')
+  end
+
+  def gte_solidus?(version)
+    Spree.solidus_gem_version >= Gem::Version.new(version)
```
Comment (kennyadsl) [inline app/helpers/spree/base_helper_decorator.rb]: I think you can use https://github.com/solidusio/solidus_support/blob/master/lib/solidus_support.rb#L7, right?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -90,9 +89,15 @@
   config.fail_fast = ENV['FAIL_FAST'] || false
   config.order = "random"
 
-  Capybara.javascript_driver = :poltergeist
-  Capybara.default_max_wait_time = 5
-  Capybara.register_driver :poltergeist do |app|
-    Capybara::Poltergeist::Driver.new(app, timeout: 1.minute, phantomjs_options: ['--load-images=no'])
+  require 'selenium/webdriver'
```
Comment (kennyadsl) [inline spec/spec_helper.rb]: what about using https://github.com/solidusio/solidus_support/blob/master/lib/solidus_support/extension/feature_helper.rb ?

Comment (DanielePalombo) [reply spec/spec_helper.rb]: It's now available in solidus_support 0.2.2 (latest solidus_support version).

**Comment by DanielePalombo** [comment]
Closing this PR because solidus 2.4 is no more supported. Keep it because it can be usefull for who wants to use this extension on solidus 2.4.

**Pull request closed without merging**